views/account/generate_qr_code/index.ejs

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en" class="h-100">
+  <head>
+    <%- include('../../partials/header'); %>
+    <title>QR Code Generator</title>
+    <link rel="stylesheet" href="/account/generate_qr_code/style" />
+    <script src="/account/generate_qr_code/main" type="module"></script>
+  </head>
+  <body class="h-100 text-center text-white bg-dark">
+    <div class="row col-12"><%- include('../../partials/navbar'); %></div>
+
+    <div
+      class="cover-container w-100 align-items-center h-75 p-3 mx-auto"
+    >
+      <div class="row col-12">
+        <main class="px-3">
+
+          <div class="form-check form-switch form-switch-xl">
+            <input class="form-check-input" type="checkbox" role="switch" id="instagram">
+            <label class="form-check-label" for="instagram">Instagram</label>
+          </div>
+          <div class="form-check form-switch form-switch-xl">
+            <input class="form-check-input" type="checkbox" role="switch" id="facebook">
+            <label class="form-check-label" for="facebook">Facebook</label>
+          </div>
+          <div class="form-check form-switch form-switch-xl">
+            <input class="form-check-input" type="checkbox" role="switch" id="twitter">
+            <label class="form-check-label" for="twitter">Twitter</label>
+          </div>
+
+          <form id="generate-form">
+            <input
+              id="url"
+              type="url"
+              placeholder="Enter a URL"
+              class="w-full border-2 border-gray-200 rounded p-3 text-grey-dark mr-2 focus:outline-none"
+              required
+            />
+
+            <select
+              class="w-full border-2 border-gray-200 rounded p-3 text-grey-dark mr-2 focus:outline-none"
+              name="size"
+              id="size"
+            >
+              <option value="100x100">100x100</option>
+              <option value="200x200" selected>200x200</option>
+              <option value="300x300">300x300</option>
```

Comment:
> Sorry but when I was saying we should limit the options, I was thinking that it might still be good to have one of the bigger options like 700x700 too. This wouldn't stop me from approving the PR though so if you decide not to change it that's fine. Feel free to chime in here @paytonshaltis @thadakn1

I think we would just need to test on mobile to make sure that it all fits on the screen, but we can easily change these values later on if we need or after user testing starts.

Reply:
Yeah the reason why I maxed out at 300x300 is because it goes off the screen in mobile. And also, for some reason I can't go past like 500x500 with the google carts api. But yeah we'll talk about it in person about it more.